dictionary/__tests__/MessageFormatSpec.js

```diff
@@ -43,4 +43,17 @@ describe('ima.MessageFormatDictionary', function() {
 		});
 	});
 
+	describe('has method', function() {
+		it('should be return true', function() {
+			expect(dictionary.has('home.title')).toEqual(true);
+			expect(dictionary.has('home.message')).toEqual(true);
+		});
+
+		it('should be return false for non existing keys and null for non last point keys', function() {
+			expect(dictionary.has('non-exists-phrase')).toEqual(false);
+			expect(dictionary.has('home.non-exists-phrase')).toEqual(false);
+			expect(dictionary.has('home')).toEqual(null);
```

Comment:
When checking booleans and nulls, you might prefer to use `.toBe` and `.toBeNull` to be more `null`-safe.